lib/translator/common.cc

```diff
@@ -35,4 +37,20 @@ void SetDebugCallback(void (*callback)(const char*)) {
   debug_callback = callback;
 }
 
```

Comment:
Can we have an abstraction for these three lines here https://github.com/googleinterns/scsi2nvme/pull/72/files#diff-b58a8575f98a080928fdb2d78d86ca29R64-R66 . Maybe `GetPrp()` but it's also used for metadata pointer so maybe a more generic name

Reply:
How about returning reinterpret_cast<uint64_t>(alloc_pages_callback(count)); ? I can't identify any cases where we actually want a void* response. An integer value of 0 implies the pointer was null.

Reply:
yeah that works.